### ISSUE TITLE
"Ground Station" info

### ISSUE THREAD

**Issue by K4KDR** [jmason86/MinXSS_Beacon_Decoder#25]
Whether your team has a use for it or not, most projects include an identifier of some kind (such as Amateur Radio call sign) in the ground station info so that some tracking or credit can be attributed to the volunteer who is submitting telemetry to your database.

**Comment by jmason86** [comment]
Yep, I have that as a TODO in two functions within `minxss_beacon_decoder.py` right now. Planning to implement very soon.

**Comment by jmason86** [comment]
Done! This is now available in commit f4095835de4fb34d410341839aadf26cc45447d1. Check it out!

**Comment by jmason86** [comment]
Yep! The [signal is called "finishedEditing" ](https://doc.qt.io/qtforpython/PySide2/QtWidgets/QLineEdit.html#PySide2.QtWidgets.PySide2.QtWidgets.QLineEdit.editingFinished) and it triggers when the user either presses enter/return or otherwise selects some other UI element for primary focus (e.g., clicking elsewhere, hitting tab). 
Commit 999841006081d5281296e3b70f4c42d7b36c755e includes this.

**Comment by K4KDR** [comment]
Wow, VERY nice.  I deleted **input_properties.cfg** as if starting from scratch, so default Ground Station info was present.  As soon as I entered my call sign (before even entering lat/lon), the upload file name changed:

![screenshot from 2018-10-04 20-04-11](https://user-images.githubusercontent.com/19886368/46509588-9f745280-c811-11e8-80da-aa57c9fd4dd3.png)

... and a test packet continued to decode and upload properly:

![screenshot from 2018-10-04 20-05-35](https://user-images.githubusercontent.com/19886368/46509596-b61aa980-c811-11e8-905e-6c5d835e34fe.png)

(... at least I assume it's uploading correctly - would have to verify on the back-end database)
